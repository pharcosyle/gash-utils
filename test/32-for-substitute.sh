for file in `echo ar.o arscan.o`; do
    echo compiling ${file}...
done
