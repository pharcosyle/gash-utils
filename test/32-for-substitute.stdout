compiling ar.o...
compiling arscan.o...
